sdkproject/Assets/Mapbox/Unity/Location/AbstractLocationProvider.cs

```diff
@@ -5,10 +5,12 @@
 
 	public abstract class AbstractLocationProvider : MonoBehaviour, ILocationProvider
 	{
+		public Location _latestLocation;
```

Comment:
Prefer getter, and please also add to `ILocationProvider` interface. Naming conventions for public/getter should be capitalized with no preceding `_`.